modern/org/commcare/modern/database/DatabaseHelper.java

```diff
@@ -0,0 +1,153 @@
+package org.commcare.modern.database;
+
+import org.commcare.modern.models.EncryptedModel;
+import org.commcare.modern.models.RecordTooLargeException;
+import org.commcare.modern.util.Pair;
+import org.javarosa.core.services.storage.IMetaData;
+import org.javarosa.core.services.storage.Persistable;
+import org.javarosa.core.services.storage.StorageFullException;
+import org.javarosa.core.util.externalizable.Externalizable;
+
+import java.io.ByteArrayOutputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Setup of platform-agnostic DB helper functions IE for generating SQL
+ * statements, args, content values, etc.
+ *
+ * @author wspride
+ *
+ */
+public class DatabaseHelper {
+
+    public static String ID_COL = "commcare_sql_id";
+    public static String DATA_COL = "commcare_sql_record";
```

Comment:
It is kinda sketch that these aren't `final`. Can they be made `final`, or is there a reason they need to be mutated?